File: chatbot/backend/model-service/models/info.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()


class Info(Base):
    __tablename__ = "infos"

    id = Column(Integer, primary_key=True, autoincrement=True, nullable=False)
    title = Column(String, nullable=True)
    text = Column(String(4000), nullable=False)

File: chatbot/backend/app-service/models/info.js
const { DataTypes } = require("sequelize")

const { sequelize } = require("../util/database")

const Info = sequelize.define("info", {
  id: {
    type: DataTypes.INTEGER,
    autoIncrement: true,
    primaryKey: true,
    allowNull: false,
  },
  title: {
    type: DataTypes.STRING,
    allowNull: true,
  },
  text: {
    type: DataTypes.STRING(4000),
    allowNull: false,
  },
})

module.exports = Info
